packages/flutter/test/widgets/html_element_view_test.dart

```diff
@@ -37,9 +38,42 @@ void main() {
       );
     });
 
+    testWidgets('Create HTML view with PlatformViewCreatedCallback',
+        (WidgetTester tester) async {
+      final int currentViewId = platformViewsRegistry.getNextPlatformViewId();
+      final FakeHtmlPlatformViewsController viewsController =
+          FakeHtmlPlatformViewsController();
+      viewsController.registerViewType('webview');
+
+      void onPlatformViewCreatedCallBack(int id) {
+        expect(id >= 0, true);
+      }
+
+      await tester.pumpWidget(
+        Center(
+          child: SizedBox(
+            width: 200.0,
+            height: 100.0,
+            child: HtmlElementView(
+              viewType: 'webview',
+              onPlatformViewCreated: onPlatformViewCreatedCallBack,
+            ),
+          ),
+        ),
+      );
+
+      expect(
+        viewsController.views,
+        unorderedEquals(<FakeHtmlPlatformView>[
+          FakeHtmlPlatformView(currentViewId + 1, 'webview'),
+        ]),
+      );
+    });
+
     testWidgets('Resize HTML view', (WidgetTester tester) async {
       final int currentViewId = platformViewsRegistry.getNextPlatformViewId();
-      final FakeHtmlPlatformViewsController viewsController = FakeHtmlPlatformViewsController();
+      final FakeHtmlPlatformViewsController viewsController =
```

Comment:
Can you revert these formatting changes? The flutter repo does not use dart format.

Reply:
sure, I will revert it.